docs/UserGuide.md

```diff
@@ -12,6 +12,10 @@ SoConnect is a **desktop app for managing contacts and tasks**. It aims to help
 
 # Quick start
 
+This section gives you information on how to install SoConnect and how to start using it. SoConnect uses a [Command Line Interface (CLI)](#CLI) which is where [commands](#Command), which are typed in a certain [format](#Command Format), are entered to use SoConnect features.
+
+## Installation Tutorial
+
 1. Ensure you have Java `11` or above installed in your Computer.
```

Comment:
Maybe we could provide a link to a website to install java 11.
```suggestion
1. Ensure you have Java [`11`](https://www.oracle.com/java/technologies/downloads/#java11) or above installed in your Computer.
```

Reply:
Ok